src/systems/ec/implementations/react/packages/inpage-navigation/examples/Default.jsx

```diff
@@ -0,0 +1,75 @@
+/* eslint-disable import/no-extraneous-dependencies */
+import React from 'react';
+import StoryWrapper from '@ecl/story-wrapper';
+import VanillaInpageNavigation from '@ecl/ec-component-inpage-navigation';
+import demoContent from '@ecl/ec-specs-inpage-navigation/demo/data';
+import loremIpsum from 'lorem-ipsum';
+import Breadcrumb, { BreadcrumbItem } from '@ecl/ec-react-component-breadcrumb';
```

Comment:
I think that the component example should not include extra components like header and footer; that's the role of the template